app/parse.py

```diff
@@ -14,5 +33,75 @@ class Course:
     course_type: CourseType
 
 
+CSV_FIELDS_COURSE = [field.name for field in fields(Course)]
+
+
+def get_course_type(course_soup: BeautifulSoup) -> [CourseType]:
+    tags = course_soup.select(".ButtonBody_buttonText__FMZEg")
+    course_types = []
+    for tag in tags:
+
+        if tag.text == "Власний темп":
+            course_types.append(CourseType("part-time"))
+
+        if tag.text == "Повний день":
+            course_types.append(CourseType("full-time"))
+
+    return course_types
+
+
+def get_page_soup() -> BeautifulSoup:
+    page = requests.get(HOME_URL).content
+    return BeautifulSoup(page, "html.parser")
+
+
+def parse_single_course(
+    course_soup: BeautifulSoup, course_type: [CourseType]
+) -> Course:
+    return Course(
+        name=course_soup.select_one(".ProfessionCard_title__Zq5ZY").text,
+        short_description=course_soup.select_one(
+            ".typography_landingTextMain__Rc8BD"
+        ).text,
+        course_type=course_type,
+    )
+
+
 def get_all_courses() -> list[Course]:
-    pass
+    logging.info(
+        f"Started parsing at {datetime.now().strftime('%d.%m.%Y, %H:%M:%S')}"
+    )
+    page_soup = get_page_soup()
+    course_cards = page_soup.select("div.ProfessionCard_cardWrapper__JQBNJ")
+
+    all_courses = []
+
+    for i, course_card in enumerate(course_cards):
```

Comment:
Do not use single-letter variable names in your code, use more descriptive naming